src/worker/worker.ts

```diff
@@ -146,6 +146,10 @@ export class ExtensionWorker implements ExtensionWorkerTransport {
     return files.map(([project, spec]) => [project.config.name || '', spec])
   }
 
+  public getResolvedBrowserOptions(): ResolvedBrowserOptions | undefined {
+    return { ...this.ctx.config?.browser, commands: undefined }
```

Comment:
This needs to be reworked, complex configuration (e.g. custom commands) will break the extension currently.

Reply:
I think you can just rename this to `getBrowserDebugOptions` and return only needed fields